darkroomjs/darkroomjs.d.ts

```diff
@@ -0,0 +1,61 @@
+// Type definitions for darkroomjs
+// Project: https://github.com/MattKetmo/darkroomjs
+// Definitions by: Stepan Mikhaylyuk <https://github.com/stepancar>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+/// <reference path="../fabricjs/fabricjs.d.ts"/>
+
+declare class DarkRoom {
+    constructor(element: string | HTMLImageElement, options?: Options)
+    plugins: {
+        crop: any;
+        save: any;
+    }
+    sourceImage: fabric.IImage;
+    /**
+    * add event listener for fabric events
+    */
+    addEventListener: (eventName: string, callBack:()=>any)=>any
+}
+interface Options {
```

Comment:
`Options` is very popular identifier. you should use ghost module pattern. http://definitelytyped.org/guides/best-practices.html#ghost-modules